public/src/cards.js

```diff
@@ -95,6 +95,9 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (App.state.beep){
+      self.beep.play()
+    }
```

Comment:
[Window.self](https://developer.mozilla.org/en-US/docs/Web/API/Window.self) is defined, but i'd rather not use it